transport/matrix/layer.py

```diff
@@ -115,14 +116,14 @@ def light_client_onboarding_data(self, address: typing.Address) -> dict:
         server_url = client.api.base_url
         server_name = urlparse(server_url).netloc
         return {
+            "transport_mode": self.transport_type,
```

Comment:
i wish we'd call this `transport_type`. we already have transport _layer_, and the transport classes have `type` defined as well—and this matches the config code.

i think adding _mode_ to this list of terms might add noise.

Reply:
agree, i will do it

Reply:
done in next commit